scripts/jenkins/dist-feed/requirements.txt

```diff
@@ -1,5 +1,4 @@
 Jinja2>=2.10.1
 PyYAML>=5.1.2
---trusted-host ni-pypi.amer.corp.natinst.com
```

Comment:
What's the reason that we can remove this line?

Reply:
The new URI is https, thus doesn't have to be explicitly trusted.